src/app/containers/MediaPlayer/__snapshots__/index.test.jsx.snap

```diff
@@ -145,6 +153,7 @@ exports[`MediaPlayer is called correctly Calls the canonical placeholder when pl
 <div
   className="c0"
 >
+  <helmet-head />
```

Comment:
Any idea what these are?

Reply:
I could change the mock to check for children before rendering.

Reply:
Yeah, might be cleaner. Worth double checking in docs that 'children' is the only possible other prop though

Reply:
Should https://github.com/bbc/simorgh/blob/latest/src/app/containers/MediaPlayer/Metadata.jsx#L8 not return null instead of an empty helmet component?

Out of scope for this PR but might be worth changing.

Reply:
Apologies but these comments are no longer relevant because of code changes. Please see the updated PR description.